shared/api/dfx/AuthService.ts

```diff
@@ -51,30 +56,50 @@ export class Session implements ISession {
 }
 
 class AuthServiceClass {
+  // TODO: (thabrad) check if able to remove!
   private readonly session$ = new ReplaySubject<Session>()
+  private dfxApiHook: (DFXAPIContextI) | undefined
```

Comment:
Is this the React workaround, you told me about?

Reply:
yup!